tests/mod.rs

```diff
@@ -57,6 +57,49 @@ fn with_inner_test_attribute_and_test_args_and_panic(x: i8, _y: i8) {
   assert_eq!(x, 0);
 }
 
+#[test_log::test]
+#[test]
+fn with_existing_test_attribute() {}
+
+#[test_log::test]
+#[::core::prelude::v1::test]
+fn with_existing_generated_test_attribute() {}
+
+#[tokio::test]
+#[test_log::test]
+async fn with_append_test_attribute_and_async() {
+  assert_eq!(async { 42 }.await, 42)
+}
+
+#[rstest]
+#[case(-2, -4)]
+#[case(-2, -5)]
+#[test_log::test]
+fn with_append_test_attribute_and_test_args(#[case] x: i8, #[case] _y: i8) {
+  assert_eq!(x, -2);
+}
+
+#[rstest]
+#[case(-2, -4)]
+#[case(-3, -4)]
+#[test_log::test]
+// Applied to all cases, must not before `rstest`, see https://github.com/la10736/rstest/issues/210
+#[should_panic] // https://docs.rs/rstest/latest/rstest/attr.rstest.html#use-specific-case-attributes
```

Comment:
Let's use versioned URLs for the sake of persistence? https://docs.rs/rstest/0.25.0/rstest/attr.rstest.html#use-specific-case-attributes